system_strings/sysmes_text.en

```diff
@@ -467,11 +467,11 @@ Slumber
 Ordinary Homeroom
 I eat in the classroom.
 Should I go to the tea room...?
-……いや、俺の起こした現実だ。
-止まない雨
-導き
-懺悔
-雨音を聞きながら。
+...No, I created this reality.
+Unstopping Rain
```

Comment:
Unstopping means to remove a stopper from something. Unrelenting, Ceaseless, Interminable, Endless